src/general/MainMenu.cs

```diff
@@ -67,12 +70,21 @@ public class MainMenu : NodeWithInput
 
     private Button newGameButton = null!;
     private Button freebuildButton = null!;
+    private Button modsButton = null!;
 
     private Label storeLoggedInDisplay = null!;
 
     private CustomConfirmationDialog gles2Popup = null!;
     private ErrorDialog modLoadFailures = null!;
 
+    public enum Menu : uint
```

Comment:
Is there a reason this is an uint? Couldn't this be just an int, which is recommended to basically always use in C# whenever you absolutely don't need uint.